drivers/npmx/npmx_shell.c

```diff
@@ -1144,6 +1144,28 @@ static int cmd_buck_vout_select_get(const struct shell *shell, size_t argc, char
 	return 0;
 }
 
+static bool charger_charging_current_set_helper(const struct shell *shell,
+						npmx_charger_t *charger_instance, uint16_t current,
+						uint16_t *p_charging_current)
+{
+	npmx_error_t err_code = npmx_charger_charging_current_set(charger_instance, current);
+	if (!check_error_code(shell, err_code)) {
+		print_set_error(shell, "charging current");
+		return false;
+	}
+
+	err_code = npmx_charger_charging_current_get(charger_instance, p_charging_current);
+	if (!check_error_code(shell, err_code)) {
+		print_get_error(shell, "charging current");
+		return false;
+	}
+
+	if (current != *p_charging_current) {
+		shell_info(shell, "Info: Set value may be different than requested.");
```

Comment:
How about
```suggestion
		shell_info(shell, "Info: Requested value was %u but reading will return %u due to approximations.", current, *p_charging_current);
```

Reply:
Same in other functions.

Reply:
Please create a helper function for this printing.